pkgs/firehose/lib/src/github.dart

```diff
@@ -0,0 +1,182 @@
+// Copyright (c) 2023, the Dart project authors.  Please see the AUTHORS file
```

Comment:
Would it make sense to have these general classes for interacting with Github in their own package?